src/config/traefik_config.rs

```diff
@@ -0,0 +1,20 @@
+use once_cell::sync::Lazy;
+
+pub struct TraefikConfig {
+    pub fqdn: String,
+    pub network_name: String,
+}
+
+impl TraefikConfig {
+    pub fn new() -> Self {
+        return TraefikConfig {
+            fqdn: std::env::var("POMEGRANATE_FQDN").unwrap_or(String::from("localhost")),
+            network_name: std::env::var("POMEGRANATE_DOCKER_NETWORK_NAME")
+                .expect("Missing proxy network !"),
+        };
```

Comment:
May remove unnecessary keyword `return` here (and final `;` if removed)
```suggestion
        TraefikConfig {
            fqdn: std::env::var("POMEGRANATE_FQDN").unwrap_or(String::from("localhost")),
            network_name: std::env::var("POMEGRANATE_DOCKER_NETWORK_NAME")
                .expect("Missing proxy network !"),
        }
```